1] guessing game/main.py

```diff
@@ -0,0 +1,23 @@
+from random import randint as ri
+
+def run():
+    hidden_number = ri(1, 100)
+
+    while True:
+        try:
+            user_number = int(input("Guess the number: "))
+        except ValueError as err:
```

Comment:
as err je zbytečné, když s tím dále nepracuješ

Reply:
hm jo to tam nemusi byt no